tests/figures/chart/charts_component.test.ts

```diff
@@ -714,6 +714,65 @@ describe("charts", () => {
     }
   );
 
+  test("can edit chart time axis limits", async () => {
```

Comment:
move the test to the line_chart_plugin.test.ts file

Reply:
I'd create a `line_chart_components` file I think, it's very specific to line charts I think